xsd/netex_framework/netex_reusableComponents/netex_trainElement_version.xsd

```diff
@@ -552,7 +603,43 @@ Rail transport, Roads and Road transport
 			</xsd:element>
 		</xsd:sequence>
 	</xsd:group>
-	<!-- ======================================================================= -->
+	<xsd:complexType name="trainsInCompoundTrain_RelStructure">
```

Comment:
I would like to see this element to be 'moved up'. For example above _TrainInCompoundTrainGroup_, so it does not look to be removed and added.

Reply:
done